storage/engine/in_mem.go

```diff
@@ -28,7 +28,9 @@ type InMem struct {
 
 // NewInMem allocates and returns a new, opened InMem engine.
 func NewInMem(attrs roachpb.Attributes, cacheSize int64, stopper *stop.Stopper) InMem {
-	cache := NewRocksDBCache(cacheSize)
+	cache := NewRocksDBCache(cacheSize) // released when db closes
```

Comment:
"released when db closes" is not quite right - i'd remove this inline comment and only keep the paragraph below